pkg/ast/ast_operation_definition.go

```diff
@@ -67,3 +60,23 @@ func (d *Document) AddVariableDefinitionToOperationDefinition(operationDefinitio
 	d.OperationDefinitions[operationDefinitionRef].VariableDefinitions.Refs =
 		append(d.OperationDefinitions[operationDefinitionRef].VariableDefinitions.Refs, ref)
 }
+
+const (
+	alphabet = `abcdefghijklmnopqrstuvwxyz`
```

Comment:
This is already discussed in another PR: https://github.com/TykTechnologies/graphql-go-tools/pull/50